lib/Doctrine/DBAL/DriverManager.php

```diff
@@ -398,11 +395,9 @@ private static function parseDatabaseUrlScheme(array $url, array $params): array
 
             // The requested driver from the URL scheme takes precedence
             // over the default driver from the connection parameters (if any).
-            $params['driver'] = isset(self::$driverSchemeAliases[$driver])
-                // use alias like "postgres", else we just let checkParams decide later
-                // if the driver exists (for literal "pdo-pgsql" etc)
-                ? self::$driverSchemeAliases[$driver]
-                : $driver;
+            $params['driver'] = self::$driverSchemeAliases[$driver] ?? $driver;
+            // use alias like "postgres", else we just let checkParams decide later
+            // if the driver exists (for literal "pdo-pgsql" etc)
```

Comment:
What is this comment now related to?

Reply:
This comment still related to the ternary expression. Should we break the right argument of `??` into the next line or should we remove it now?

Reply:
> This comment still related to the ternary expression.

This is not obvious. Usually, comments are added prior to the code they describe but this commit changes the order of the comment and code.

> Should we break the right argument of `??` into the next line or should we remove it now?

I believe it should be fine to keep it above the line with the code. Both `The requested driver from the URL […]` and `use alias like "postgres" […]` describe the same single line of code.

Reply:
Can I suggest:
> The requested driver from the URL scheme takes precedence over the default driver from the connection parameters. If any, use alias like "postgres", else we just let checkParams decide later if the driver exists (for literal "pdo-pgsql" etc).

Reply:
How about this:

> […] If the driver is an alias (e.g. "postgres"), map it to the actual name ("pdo-pgsql"). Otherwise, let checkParams decide later if the driver exists.

:+1: to combining the comments.

Reply:
Amending the previous commit with this comment update :man_technologist: